app/models/extensions/searchable.rb

```diff
@@ -43,9 +43,12 @@ def search_cfg
       end
     end
 
-    def joined_resource_fields(fields)
-      fields ||= []
-      fields.map { |prop| processed_resource[prop] }.compact.join("\n")
+    def joined_resource_fields(paths)
+      (paths || []).map do |path|
+        value = JsonPath.on(processed_resource_before_type_cast, path).first
+        next if value.blank?
+        value.gsub(/[:?]/, ' ')
```

Comment:
this it's just removing ':', right? but why? could you a comment.

Reply:
This replaces colons and question marks with blank spaces. Without that, URLs are not vectorized correctly.